utils.py

```diff
@@ -456,18 +456,39 @@ def unescape_encoded_uri_component(escaped_string):
     return urllib.unquote(escaped_string).decode('utf-8')
 
 
-ASSET_DIR_PREFIX = None
+_CACHE_SLUG = None
+def get_cache_slug():
+    """Returns cache slug depending whether dev or prod."""
+    global _CACHE_SLUG # pylint: disable=global-statement
```

Comment:
Why wouldn't having it as a module-level variable cache it? The variable would exist for each instance of ``utils`` that's imported, but this isn't very frequent. Beyond that, it only performs the operation if this function is called. If you add a print statement, does it actually redefine ``_CACHE_SLUG`` more than a couple of times across multiple browser sessions? I can't imagine it does. If so, it's worth keeping as a global. If not, we really ought to not do that.